variables.tf

```
@@ -0,0 +1,23 @@
+variable "aws_account_alternate_contacts" {
+  type = object({
+    Billing = optional(object({
+        name = ""
+        title = ""
+        email_address = ""
+        phone_number = ""
+    }))
```

Let's lowercase these object attribute names (matching how we reference the attribute in `main.tf`), and sort the attributes alphabetically.

Also, in the `type` value, to indicate an input accepts string-types, use the keyword `string` instead of the empty string value `""`.

Oh, and run `terraform fmt -recursive` to auto-format the files.

Here's the resulting change to this block. I'll let you make the change to the other two.

```suggestion
    billing = optional(object({
        email_address = string
        name          = string
        phone_number  = string
        title         = string
    }))
```